zpl_generator.py

```
@@ -488,3 +488,78 @@ def generate_tracescan_label(
 
     ^XZ
     """
+
+
+def generate_svt_fortlox_label(
+    printer_id: str,
+    sv_article_no: str,
+    serial_no: str,
+    fw_version: str,
+    run_date: str,
+    ) -> str:
```

Flake8 (7.2.0)

[error] continuation line with same indent as next logical line

(E125)